test/client_and_receiver_integration_test.exs

```diff
@@ -523,15 +524,27 @@ defmodule ClientAndReceiverIntegrationTest do
     @tag verify: :verify_none
     @tag client_cert: ""
     @tag keyfile: ""
+    @tag reason: [{:options, {:certfile, ""}}]
     test "does not verify client cert if verify none option is provided on receiver", ctx do
-      make_call_and_assert_success(ctx, ctx.ack)
+      if otp_release() < 26 do
+        make_call_and_assert_success(ctx, ctx.ack)
+      else
+        make_call_and_assert_failure(ctx, ctx.reason)
+      end
     end
 
     @tag port: 8162
     @tag client_cert: ""
     @tag keyfile: ""
 
-    @tag reason: [:handshake_failure, :certificate_required]
+    # @tag reason: if otp_release() < 26 do
```

Comment:
Not needed or not needed right now?

Reply:
Oh, that was my attempt to solve it with @tag.
It didn't work, as @tag apparently does not take expressions.
Would be cool if it did :-)
Not needed, as it's covered by the @tag below.